src/app/meta-campaign/meta-campaign.component.scss

```diff
@@ -15,3 +15,58 @@
 .mat-spinner {
   margin: 3rem auto 0;
 }
+
+.align-center {
+  display: block;
+  margin: auto;
+}
+
+.padding-top-s {
+  padding-top: 2rem;
+}
+
+.padding-top-m {
```

Comment:
I'm not sure about adding lots more utility classes, especially ones that aren't used and especially when the element already has a more descriptive class applied to it. This all adds to the bundle size.